src/main/java/duke/Duke.java

```diff
@@ -1,3 +1,5 @@
+package duke;
```

Comment:
Might want to sort your imports alphabetically (i.e. `duke.command`, then `duke.task`, and finally `duke.other`), though honestly, this isn't very critical (or you can even let IntelliJ sort it for you)!